Week2/homework/maartjes-work.js

```diff
@@ -44,18 +44,19 @@ const tuesday = [
 
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
-
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  let changeToHours = tasks.map((elem, index, arr) => (arr[index].duration) / 60);
+  let greaterThanTwoHours = changeToHours.filter((elem) => elem > 2);
+  let bills = greaterThanTwoHours.map((elem) => elem * hourlyRate);
+  let totalBill = bills.reduce((acc, elem) => acc + elem);
+  return totalBill;
```

Comment:
you can chain these operators together, but you made intermittent variables, with good variable names. Nice!

Reply:
chained ! thank you.